spark-scala/src/test/scala/com/matthewrathbone/spark/SparkJoinsTest.scala

```diff
@@ -0,0 +1,36 @@
+package test.scala.com.matthewrathbone.spark
+
+import org.scalatest.junit.AssertionsForJUnit
+import org.junit.Before
+import org.apache.spark.SparkConf
+import org.apache.spark.SparkContext
+import org.junit.Test
+import scala.collection.Map
+import org.apache.spark.SparkContext._
+
+class SparkJoinsTest extends AssertionsForJUnit {
+  
+  var sc: SparkContext = _
+  
+  @Before def initialize() {
+	 val conf = new SparkConf().setAppName("SparkJoins").setMaster("local")
+	 sc = new SparkContext(conf)
+  }
+  
+  @Test def sortByKey() {
+    val transactions: List[Tuple2[Int, Int]] = List(new Tuple2[Int, Int](1, 1), new Tuple2[Int, Int](2, 1),
```

Comment:
so for example this could be:

`List( (1, 1), (2, 1), (2, 1), (3, 2), (3, 1) )`